src/Workleap.Extensions.OpenAPI/Builder/JsonSerializerDifferenceException.cs

```diff
@@ -0,0 +1,3 @@
+namespace Workleap.Extensions.OpenAPI.Builder;
+
+public sealed class JsonSerializerDifferenceException(string propertyName) : Exception($"JsonSerializerOptions for {propertyName} is different.");
```

Comment:
I'm putting myself into the shoes of a dev without knowledge of this behavior, and I think we could make this error more actionable or understandable, like:

> The JSON serializer option values for '{propertyName}' differ between Microsoft.AspNetCore.Mvc.JsonOptions and Microsoft.AspNetCore.Http.JsonOptions.

It's a bit more verbose, but it provides more context and makes devs realize that there are two separate options for JSON serialization in ASP.NET Core. They now know where to look at the existing values.